src/browser/index.html

```diff
@@ -4,6 +4,26 @@
 <head>
   <meta charset="UTF-8">
   <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.min.css">
+  <style>
+
+    /*
+     * Note:
+     *
+     * Unusual placement of these styles is so that they activate while the loading
+     * screen is up rather than applying after React initializes to prevent a flash
+     * of unstyled content.
+     *
+     */
+
+    .TabBar .teamTabItem span {
+      display: inline-block;
+      font-weight: bold;
```

Comment:
`font-weight: bold` must be used only in the case where the guest page has unread channels.